cmds/core/netcat/netcat.go

```diff
@@ -87,8 +87,12 @@ func evalParams(args []string, f flags) (*netcat.Config, error) {
 	// The port is optional as the second argument.
 	switch config.ConnectionMode {
 	case netcat.CONNECTION_MODE_CONNECT:
+		if f.sourcePort == "" {
```

Comment:
You are in a sense recreating the default value you took out in the declaration below, I am wondering why.